enforcer-parent/enforcer/src/main/resources/docker-entrypoint.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+# --------------------------------------------------------------------
+# Copyright (c) 2024, WSO2 LLC. (http://wso2.com) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
If having a script isn't mandatory let's remove it. If the only purpose the script serves is adding a delay, let's do it through the pre-stop hook rather than writing a script.

Reply:
Ack removed.